### PR TITLE
System/Load: Handle PSP Remaster

### PR DESCRIPTION
**Pull request by AniLeo**
This just makes RPCS3 execute PSP Remaster games through psp_emulator.self

pspemu does not work yet, so the boot will fail, but at the least it starts loading

### CODE REVIEW

File: rpcs3/Emu/System.cpp [hunk 1]
```diff
@@ -1160,6 +1160,17 @@ game_boot_result Emulator::Load(const std::string& title_id, bool add_only, bool
 			fs::file card_2_file(vfs::get("/dev_hdd0/savedata/vmc/" + argv[2]), fs::write + fs::create);
 			card_2_file.trunc(128 * 1024);
 		}
+		else if (m_cat == "PE" && from_hdd0_game)
```
Comment (AniLeo) [inline rpcs3/Emu/System.cpp]: All PE games are digital by the way. The physical versions just have packages for the digital versions.

File: rpcs3/Emu/System.cpp [hunk 2]
```diff
@@ -1160,6 +1160,17 @@ game_boot_result Emulator::Load(const std::string& title_id, bool add_only, bool
...
+		{
+			// PSP Remaster located in dev_hdd0/game
+			sys_log.notice("PSP Remaster Game: %s, %s", m_title_id, m_title);
+
+			std::string gamePath = m_path.substr(m_path.find("/dev_hdd0/game/"), 24);
```
Comment (Megamouse) [inline rpcs3/Emu/System.cpp]: ugly copy pasta
```suggestion
			const std::string game_path = m_path.substr(m_path.find("/dev_hdd0/game/"), 24);
```

File: rpcs3/Emu/System.cpp (hunk 1 above)
Comment (JohnHolmesII) [reply rpcs3/Emu/System.cpp]: Perhaps this comment is best inlined in the code. There are far too many magic strings and numbers floating around as it is.

Comment (Megamouse) [reply rpcs3/Emu/System.cpp]: It's kinda obvious though, and tbh the comment confuses more than it helps

Comment (JohnHolmesII) [reply rpcs3/Emu/System.cpp]: It is not obvious, and the comment does not confuse more than it helps. It is non-trivial that all those games are digital; it is quite reasonable to wonder if this were an assumed hack. Having that errata tied to the code saves time in the future.

Comment (Megamouse) [reply rpcs3/Emu/System.cpp]: What should they else be?
It's physically impossible to put a umd into the ps3

Comment (Megamouse) [reply rpcs3/Emu/System.cpp]: And theres literally a category for physical disc games.

Comment (AniLeo) [reply rpcs3/Emu/System.cpp]: PSP Remasters are distributed physically on PS3 BDs, while PSP Minis are distributed digitally on PSN, that's why I made the comment

**Pull request merged**